_includes/frontpage.html

```diff
@@ -50,13 +50,13 @@ <h1>Functional Relational Mapping for Scala</h1>
 						compiler which can generate code for different backends.</p>
 					<div class="box">
 						<div class="bg">
-							<video src="http://downloads.lightbend.com/videos/hello-slick.mp4" width="600" height="338" poster="{{ site.baseurl }}/resources/images/hello-slick.png"></video>
+							<video src="https://downloads.lightbend.com/videos/hello-slick.mp4" width="600" height="338" poster="{{ site.baseurl }}/resources/images/hello-slick.png"></video>
 						</div>
 					</div>
 				</div>
 				<div class="span4">
-					<h1><a href="http://lightbend.com/activator/template/hello-slick-3.1">New to Slick?</a></h1>
-					<p>Get started in minutes using the <a href="http://lightbend.com/activator/template/hello-slick-3.1">Hello Slick</a> template in <a href="http://lightbend.com/activator">Lightbend Activator</a>.</p>
+					<h1><a href="https://lightbend.com/activator/template/hello-slick-3.1">New to Slick?</a></h1>
+					<p>Get started in minutes using the <a href="https://lightbend.com/activator/template/hello-slick-3.1">Hello Slick</a> template in <a href="https://lightbend.com/activator">Lightbend Activator</a>.</p>
```

Comment:
The Activator is long-gone now, we should link to https://developer.lightbend.com/start/?group=slick

Reply:
Good to know 👍 I updated both links